hack/rapture/README.md

```diff
@@ -103,7 +103,15 @@ The entire build process takes several hours. Once you are ready to begin, the d
 
 ### Validating packages
 
-Now that `rapture` has successfully complete, we need to verify the packages that were just created. This validation can be done on any instance where Kubernetes is not already installed. (Ideally, you would want to spin up a new VM to test.)
+Now that `rapture` has successfully complete, we need to verify the packages that were just published and created correctly.
+
+To check for publish success, see the [Debian package list][deb-package-list] and [RPM package list][rpm-package-list] for the versions that were just uploaded. Or curl on cmdline for Debian and RPM respectively:
+```
```

Comment:
nit:  ` ```shell `

Reply:
can we wrap this up into a script command that takes the version instead?
the less manual hackery the better.

Reply:
Yep I can add a quick one